.github/workflows/docker-tests.yml

```diff
@@ -9,33 +9,13 @@ on:
 
 jobs:
   build-on-linux:
-    strategy:
-      matrix:
-        docker-version: [19.03]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     name: Build and run Docker images
     steps:
-      - name: Update Packages
-        run: sudo apt-get update -yqq --fix-missing
-      - name: Install Docker
-        uses: docker-practice/actions-setup-docker@master
-        with:
-          docker_version: ${{ matrix.docker-version }}
-          docker_buildx: false
-      - name: Cache docker
-        uses: actions/cache@v1
-        with:
-          path: ~/.docker
-          key: ${{ runner.os }}-docker-${{ hashFiles('**/Dockerfile') }}
-          restore-keys: ${{ runner.os }}-docker
       - name: Checkout Repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 10
```

Comment:
Given the comment on the other PR, any reason for this to be 10?

Reply:
I guess we discussed this here: https://github.com/hypertrace/attribute-service/pull/43#discussion_r543515520 so changing this to 0 for now.